packages/boutique/backend/Dockerfile.prod

```diff
@@ -24,10 +25,12 @@ WORKDIR /testnet
 COPY package.json pnpm-lock.yaml pnpm-workspace.yaml .npmrc .nvmrc tsconfig.base.json  ./
 COPY packages/boutique/backend ./packages/boutique/backend
 COPY packages/shared/backend ./packages/shared/backend
+COPY packages/boutique/shared ./packages/boutique/shared
 
 RUN pnpm boutique:backend build
 RUN rm -rf ./node_modules ./packages/boutique/backend/node_modules
 RUN rm -rf ./node_modules ./packages/shared/backend/node_modules
+RUN rm -rf ./node_modules ./packages/boutique/shared/node_modules
```

Comment:
Why are we removing the root `node_modules` 3 times. Doing `RUN rm -rf ./node_modules ./packages/boutique/backend/node_modules ./packages/shared/backend/node_modules ./packages/boutique/shared/node_modules` is breaking something?